cmd/query/app/static_handler.go

```diff
@@ -105,17 +113,11 @@ func loadAndEnrichIndexHTML(open func(string) (http.File, error), options Static
 		return nil, fmt.Errorf("cannot load index.html: %w", err)
 	}
 	// replace UI config
-	configString := "JAEGER_CONFIG = DEFAULT_CONFIG"
-	if config, err := loadUIConfig(options.UIConfigPath); err != nil {
+	if configObject, err := loadUIConfig(options.UIConfigPath); err != nil {
 		return nil, err
-	} else if config != nil {
-		// TODO if we want to support other config formats like YAML, we need to normalize `config` to be
-		// suitable for json.Marshal(). For example, YAML parser may return a map that has keys of type
-		// interface{}, and json.Marshal() is unable to serialize it.
-		bytes, _ := json.Marshal(config)
-		configString = fmt.Sprintf("JAEGER_CONFIG = %v", string(bytes))
+	} else if configObject != nil {
+		indexBytes = configObject.regexp.ReplaceAll(indexBytes, append([]byte(`${1}`), configObject.config...))
```

Comment:
I am confused by `${1}` - is it just repeating whitespace? If we match on `// JAEGER...` and replace just that, the whitespace becomes irrelevant.

Reply:
you are correct, it's for white spaces for function UIConfig(). To save origin indent 😶 . Will remove it.